Monika After Story/game/script-affection.rpy

```diff
@@ -938,23 +945,26 @@ init 15 python in mas_affection:
             "...Fine.",
             "I guess we can play a game.",
             "I guess, if you really want to.",
-            "I suppose a game would be fine."
+            "I suppose a game would be fine.",
+            "...Yes...why not?"
         ]
         save_quips(DISTRESSED, quips)
 
         ## UPSET quips
         quips = [
             "...Which game?",
             "Okay...whatever, choose a game.",
-            "Fine, pick a game."
+            "Fine, pick a game.",
+            "Ok, choose game already."
         ]
         save_quips(UPSET, quips)
 
         ## NORMAL quips
         quips = [
             "What would you like to play?",
             "What did you have in mind?",
-            "Anything specific you'd like to play?"
+            "Anything specific you'd like to play?",
+            "What would you like to play?"
```

Comment:
the first line here is already "What would you like to play?", are you intending to increase the chances of this one? 
Not really any solid reasons for increasing chance of a particular line.